src/main/java/com/icthh/xm/tmf/ms/resource/ResourceApp.java

```diff
@@ -19,6 +14,12 @@
 import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
 import org.springframework.core.env.Environment;
 
+import javax.annotation.PostConstruct;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.Arrays;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version